pkg/api/handlers.go

```diff
@@ -291,6 +291,7 @@ func NewQuerierHandler(
 	seriesQueryStats := usagestats.NewRequestsMiddleware("querier_series_query_requests")
 	metadataQueryStats := usagestats.NewRequestsMiddleware("querier_metadata_query_requests")
 	cardinalityQueryStats := usagestats.NewRequestsMiddleware("querier_cardinality_query_requests")
+	otherStats := usagestats.NewRequestsMiddleware("querier_other_requests")
```

Comment:
It felt like it doesn't fit to any other stats classes, but not sure if others is ok? #naming

Reply:
All of the other statistics instances are specific, so I think we can be specific here as well: something like `querier_formatting_requests` perhaps?

Reply:
Ok, renamed as suggested 08178f8ba